tests/integration/targets/netaddr_test/vars/main.yaml

```diff
@@ -3,3 +3,9 @@ networks:
   - "10.0.0.0/8"
   - "192.168.1.0/24"
   - "172.16.0.0/16"
+
+good_name: www.google.com
+bad_name: foo.google.com
```

Comment:
It doesn't look like you're using these?

Reply:
Updated.
The vars are being used in the integration test for 'resolvable' test plugin.